library/build.gradle

```diff
@@ -9,8 +10,8 @@ android {
     defaultConfig {
         minSdkVersion 9
         targetSdkVersion 25
-        versionCode 1
-        versionName "1.0"
+        versionCode gitVersionCode
+        versionName gitVersionName
```

Comment:
You know... libraries don't really need versionCode and versionName fields because they're ignored, you can just remove them and be happy. That would also avoid the need of calculating a version code because it's not needed.